app/lib/widgets/content/view/file-tile.component.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:pipercrux/widgets/content/model/content.model.dart';
+import 'package:provider/provider.dart';
+import 'package:line_awesome_icons/line_awesome_icons.dart';
+
+class FileTileComponent extends StatelessWidget {
```

Comment:
Component suffix is kinda redundant, but may be not.

Reply:
that means it's a small widget which is just a part of some widget with 'view' suffix